app/src/main/java/com/asfoundation/wallet/ui/iab/InAppPurchaseInteractor.java

```diff
@@ -430,7 +430,7 @@ private boolean shouldSwap(List<PaymentMethod> paymentMethods, int position) {
       } else if (paymentMethod.getGateway() != null && (paymentMethod.getGateway()
           .getName() == (Gateway.Name.myappcoins)
           || paymentMethod.getGateway()
-          .getName() == (Gateway.Name.adyen_v2)) && isUnavailable(paymentMethod)) {
+          .getName() == (Gateway.Name.adyen_v2)) && paymentMethod.isAvailable()) {
```

Comment:
Did you forgot to put "!" here? We were checking for unavailability here and now we are checking for availability

Reply:
Fixed